point_viewer_proto_rust/src/proto.proto

```diff
@@ -119,13 +119,28 @@ message S2Cell {
   int64 num_points = 2;
 }
 
+message OctreeMeta {
+  AxisAlignedCuboid bounding_box = 1;
+  double resolution = 2;
+  repeated OctreeNode nodes = 3;
+}
+
+message S2Meta {
+  repeated S2Cell cells = 1;
+  repeated Attribute attributes = 2;
+}
```

Comment:
Should we also have a `level` field?

Reply:
I would add it, once we need it... can you already think of a use case at the moment?

Reply:
the level can be also indirectly given from the s2 cell name, which should be unique up to level 30.